src/dvclive/dvc.py

```diff
@@ -77,12 +77,16 @@ def make_checkpoint():
 def get_dvc_repo():
     from dvc.exceptions import NotDvcRepoError
     from dvc.repo import Repo
-    from dvc.scm import SCMError
+    from dvc.scm import Git, SCMError
+    from scmrepo.exceptions import SCMError as GitSCMError
```

Comment:
I am a little confused by the 2 SCMError, I am guessing one inherits from DvcException and the other not?

Reply:
Yes. This felt not too clean to me but didn't immediately know a better way to handle it.